Libplanet.Tools/package.json

```diff
@@ -8,24 +8,23 @@
     "linux",
     "win32"
   ],
-  "cpu": [
-    "x64"
-  ],
+  "cpu": ["x64"],
   "files": [
     "README.md",
-    "bin/planet",
-    "download.js",
-    "install.js"
+    "bin/planet.mjs",
+    "download.mjs",
+    "install.mjs"
```

Comment:
Changing the extensions to `.mjs` is a nice touch, but I just wanted to point out that this isn't required. When `"type": "module"` is specified, Node.js should treat all content as ES Modules unless it has a `.cjs` extension.

Reply:
That's good to know!  However, as like me, most Libplanet maintainers are not that familiar to npm, IMHO it would be better to leave them explicit.